code/game/mecha/mecha_parts.dm

```diff
@@ -218,11 +218,11 @@
 	emote = e
 
 /obj/item/mecha_parts/chassis/reticence/hear_message(mob/living/M, msg)
```

Comment:
Ironically, this being `hear_message()` means that the mime has to make an actually audible emote for it to register at all, meaning they need to break their vow for the promise of ultimate power.
That's something to fix in another PR though.

Reply:
Directing emotes at the mech doesn't work either because of this but surely there's a way around it. I currently have the vaguest idea of how that may work but I'm not nearly experienced enough to do it. The only emote that the mime can use to solo-build this mech is the "snap" emote as it's not silent for mimes.

Reply:
Currently "Visible" messages only get shown to mobs, everything else is just skipped over. "Audible" messages seem to avoid that though so in theory it should be possible to just copy that system over.
https://github.com/ParadiseSS13/Paradise/blob/f8a297b6158ec6fcf02fd02d7bc58951ac34165b/code/modules/mob/mob.dm#L108-L115
https://github.com/ParadiseSS13/Paradise/blob/f8a297b6158ec6fcf02fd02d7bc58951ac34165b/code/modules/mob/mob.dm#L133-L153
Again though, changing that in this PR is probably out of scope since the actual mech construction works fine, it just has some wierdness for the mime one.

Reply:
fart works fine